atdd.opam

```diff
@@ -0,0 +1,85 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "DLang code generation for ATD APIs"
+description: "DLang code generation for ATD APIs"
+maintainer: [
+  "Louis Roché <louis@louisroche.net>"
+  "Martin Jambon <martin@mjambon.com>"
+  "Rudi Grinberg <me@rgrinberg.com>"
+]
+authors: [
+  "Martin Jambon <martin@mjambon.com>"
+  "Rudi Grinberg <rudi.grinberg@gmail.com>"
+  "Martin Jambon <martin@r2c.dev>"
+  "Martin Jambon <github@mjambon.com>"
+  "Ivan Jager <aij+git@mrph.org>"
+  "oleksiy <oleksiy.golovko@ahrefs.com>"
+  "David Sheets <sheets@alum.mit.edu>"
+  "Rudi Grinberg <me@rgrinberg.com>"
+  "Martin Jambon <martin@esper.com>"
+  "Jeff Meister <nanaki@gmail.com>"
+  "Caio Wakamatsu <caio.wakamatsu@ahrefs.com>"
+  "Carmelo Piccione <carmelo.piccione@gmail.com>"
+  "Daniel Weil <danweil68@gmail.com>"
+  "Egor Chemokhonenko <egor.chemohonenko@ahrefs.com>"
+  "Gabriel Scherer <gabriel.scherer@gmail.com>"
+  "Raman Varabets <roman.vorobets@gmail.com>"
+  "tzm <frank@boldsolutions.de>"
+  "Mathieu Baudet <mathieubaudet@fb.com>"
+  "Oleksiy Golovko <alexei.golovko@gmail.com>"
+  "Rauan Mayemir <rauan@mayemir.io>"
+  "Carmelo Piccione <cep1@solvuu.com>"
+  "John Billings <john@monkeynut.org>"
+  "Louis Roché <louis@louisroche.net>"
+  "Brendan Long <self@brendanlong.com>"
+  "Chris Yocum <cyocum@gmail.com>"
+  "Louis Roché (Ahrefs) <louis.roche@ahrefs.com>"
+  "Louis Roché <louis.roche@ahrefs.com>"
+  "Pavel Antoshkin <pavel.antoshkin@ahrefs.com>"
+  "Pierre Boutillier <pierre.boutillier@laposte.net>"
+  "Shon Feder <shon.feder@key.me>"
+  "Anurag Soni <anuragsoni.13@gmail.com>"
+  "Arjun Ravi Narayan <arjunravinarayan@gmail.com>"
+  "Asya-kawai <kawai-toshiki@aintek.xyz>"
+  "Christophe Troestler <christophe.Troestler@umons.ac.be>"
+  "Damien Doligez <ddoligez@janestreet.com>"
+  "Daniel M <dan.mntg@gmail.com>"
+  "Ding Xiang Fei <dingxiangfei2009@protonmail.ch>"
+  "François Pottier <francois.pottier@inria.fr>"
+  "Javier Chavarri <javier.chavarri@gmail.com>"
+  "Kate <kit.ty.kate@disroot.org>"
+  "Louis <louis.roche@ahrefs.com>"
+  "Louis Roché <louis@cryptosense.com>"
+  "Raman Varabets <raman+git@ahrefs.com>"
+  "Stephane Legrand <slegrand45@gmail.com>"
+  "Vincent Bernardoff <vb@luminar.eu.org>"
+  "haoyang <haoyang@esper.co>"
+  "pmundkur <prashanth.mundkur@gmail.com>"
+  "ygrek <ygrek@autistici.org>"
+]
+license: "MIT"
+homepage: "https://github.com/ahrefs/atd"
+bug-reports: "https://github.com/ahrefs/atd/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "atd" {>= "2.11.0"}
+  "cmdliner" {>= "1.1.0"}
+  "re"
+  "alcotest" {with-test}
+  "odoc" {with-doc}
```

Comment:
@elrandar I noticed that you updated this file by hand to remove `alcotest` and to add `ldc2`. This file is generated, it shouldn't be updated by hand. Check `dune-project` instead.